101-traffic-manager-virtual-machines/azuredeploy.json

```diff
@@ -0,0 +1,271 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
```

Comment:
True but think about it from the perspective of the user who is starting to learn about IaaS. Asking them to learn about complex VM scenarios in 101 seems a bit much :) Which is why we usually reserve more complex VM scenarios for 201